### PR TITLE
Adding Paging functionality + sort

### PR DESCRIPTION
**Pull request by AlexTurff**
Added logic to page post requests to retrieve the entire post history. Added "Old" sort to ensure chronological order

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -45,38 +46,55 @@ def get_lemmy(username: str, password: str) -> Lemmy:
...
     old = get_old_lemmy()
     new = get_lemmy(config["Current"]["username"], config["Current"]["password"])
+    postSort = SortType.Old
+    postCounter = 1
+    pageCounter = 1
     if not args.profile:
         print("Will migrate communities.")
         old_com = old.discover_community(config["Community Migration"]["old"])
         old_posts = old.post.list(
-            old_com, limit=int(config["Community Migration"]["limit"])
```
Comment (taskylizard) [inline migrate.py:52]: Remove this from config.ini.example if we won't use it anymore.

File: migrate.py [hunk 2]
```diff
@@ -45,38 +46,55 @@ def get_lemmy(username: str, password: str) -> Lemmy:
...
+                        )
+                postCounter+=1
+            pageCounter+=1
+            try:
+                old_posts = old.post.list(old_com,sort=postSort,page=pageCounter)
+            except Exception as e:
+                old_posts = []
+                print(f"fetch error: {e}")
+                break
+                break
```
Comment (taskylizard) [inline migrate.py:97]: Mistype probably but two `break`s for?

**Review by taskylizard — CHANGES_REQUESTED** [review]

File: migrate.py (hunk 1 above)
Comment (AlexTurff) [reply migrate.py:52]: Done

File: migrate.py (hunk 2 above)
Comment (AlexTurff) [reply migrate.py:97]: Done

**Review by taskylizard — APPROVED** [review]

**Comment by taskylizard** [comment]
Thanks for the PR!

**Pull request merged**